<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DesignTipDialog</class>
 <widget class="QDialog" name="DesignTipDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Did you know that ... </string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="whatsThis" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="0" >
    <widget class="QCheckBox" name="AgainBox" >
     <property name="text" >
      <string>Show tips on startup!</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="5" >
    <widget class="QPushButton" name="DoneButton" >
     <property name="text" >
      <string>Ok</string>
     </property>
     <property name="icon" >
      <iconset>image1</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="4" >
    <widget class="QPushButton" name="NextButton" >
     <property name="text" >
      <string>Next</string>
     </property>
     <property name="icon" >
      <iconset>image2</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>41</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>71</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>91</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>11</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="font" >
        <font>
         <family></family>
         <pointsize>24</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Did you know that ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="6" >
    <widget class="QTextBrowser" name="TipEdit" />
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
